Source/WebKit/WebProcess/GPU/webrtc/RemoteVideoFrameObjectHeapProxyProcessor.cpp

```diff
@@ -39,15 +39,16 @@ namespace WebKit {
 
 Ref<RemoteVideoFrameObjectHeapProxyProcessor> RemoteVideoFrameObjectHeapProxyProcessor::create(GPUProcessConnection& connection)
 {
-    return adoptRef(*new RemoteVideoFrameObjectHeapProxyProcessor(connection));
+    auto processor = adoptRef(*new RemoteVideoFrameObjectHeapProxyProcessor(connection));
+    processor->initialize();
```

Comment:
I keep doing this error, let's hope this is the last time.
Could we do something like RefCounted so that addWorkQueueMessageReceiver would debug assert that the receiver was adopted-ref?

Reply:
It's an interesting idea. I'll try to think about it -- it's a bit tricky as the WorkQueueMessageReceiver probably should itself be removed, at it is not very useful class and it tends to create a circular reference (as in this case).